libc/src/math/generic/log1pf.cpp

```diff
@@ -57,7 +57,7 @@ LIBC_INLINE float log(double x) {
   // Get the 8 highest bits, use 7 bits (excluding the implicit hidden bit) for
   // lookup tables.
   int f_index = static_cast<int>(
-      xbits.get_mantissa() >> 45); // fputil::MantissaWidth<double>::VALUE - 7
+      xbits.get_mantissa() >> 45); // fputil::FPBits<double>::MANTISSA_WIDTH - 7
```

Comment:
Nit: let's replace `45` with the comment.

Reply:
(possibly in a separate commit)

Reply:
Done